src/modules/ekf2/EKF2.cpp

```diff
@@ -1333,6 +1336,21 @@ void EKF2::publish_wind_estimate(const hrt_abstime &timestamp)
 	}
 }
 
+void EKF2::publish_optical_flow_vel(const hrt_abstime &timestamp)
+{
+	optical_flow_vel_s flow_vel{};
+	flow_vel.timestamp_sample = timestamp;
+
+	_ekf.getFlowVelBody().copyTo(flow_vel.vel_body);
```

Comment:
Could this be similar to `vehicle_visual_odometry_aligned` where it only publishes in the same cycle that new data was received, but after the EKF update?

Reply:
Not sure if it's worth doing this because the optical flow update rate is probably faster than the EKF update rate

Reply:
It's more of a log visibility (and minor optimization) thing, only having `estimator_optical_flow_vel` when you actually had optical_flow and were using it.

Getting the optical_flow data rate within the EKF update rate is one of the other reasons I was going to pipe it through the sensors module first.